ajax/libs/epub.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "epub.js",
+  "filename": "epub.min.js",
+  "description": "Render ePub documents in the browser, across many devices",
+  "keywords": [
+    "epub",
+    "ebooks"
+  ],
+  "author": "fchasen@gmail.com",
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/futurepress/epub.js.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "+(epub|hooks|reader)*",                                                                                                                                                            
+          "!(libs)/**/*" 
+        ]
+      }
+    ]
+  },
+  "repository": {
```

Comment:
If this field can be moved under `author` field, this will be a great work.
Because this would help people to find this repo quickly.
What do you think?

Thank you.